libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/connectionsTab.tsx

```diff
@@ -20,9 +21,10 @@ export const connectionsTab = (intl: IntlShape, dispatch: AppDispatch, nextTabId
     description: 'The tab label for the monitoring connections tab on the create workflow panel',
   }),
   description: intl.formatMessage({
-    defaultMessage: 'Connections Tab',
-    id: 'hsZ7em',
-    description: 'An accessability label that describes the connections tab',
+    defaultMessage:
+      'Configure connections to authenticate the following services and link your workflows with various services and applications, enabling seamless data integration and automation. Connections are required.',
+    id: '2zqm3d',
+    description: 'An accessability label that describes the objective of connections tab',
```

Comment:
I thought I had this covered in the displayConnections component

Reply:
I see it, I will be removing it from displayConnections component. This description folows the same format and styling as other tabs, so it'd be better to keep a consistent way for all.